modules/devguide/examples/nodejs/migrating.js

```diff
@@ -104,20 +91,16 @@ async function certauth(){ // use one in  connecting-cert-auth.js
       // #end::certauth[]
 }
 
-async function simpleget(){
-      shutdown();
-      console.log("simpleget:");
-      // #tag::simpleget[]
-      const options = { username:"Administrator", password:"password"};
-      cluster = new couchbase.Cluster( "http://127.0.0.1", options);
-      bucket = cluster.bucket("travel-sample");
-      collection = bucket.defaultCollection();
-      const getResult = await collection.get("airport_1254");
-      console.log(getResult);
-      cluster.close();
-      // #end::simpleget[]
-      connect();
-    }
+async function simpleget() {
+  console.log("simpleget:");
+  
+  // tag::simpleget[]
+  
+  const getResult = await collection.get("airport_1254");
+  
+  // end::simpleget[]
```

Comment:
Don't think we need the extra lines here.

Reply:
this was deliberate for formatting I think. (Antora's `include[tags...]` macro annoyingly not letting you add additional content between includes in the same source block.